ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py

```diff
@@ -17,18 +17,17 @@ def test_invalid_user_login(self):
         account.input_login_username('invalid')
         account.input_login_password('pass123')
         account.click_login_button()
-        # import pdb; pdb.set_trace()
 
         #Validate the actual message is same as the expected error message.
         expected_msg="Error: The username invalid is not registered on this site. If you are unsure of your username, try your email address instead."
         account.wait_until_error_is_displayed(expected_msg)
 
-
+        # TODO: Checkout what this is
         # account.get_login_form_heading()

```

Comment:
just delete this block. It does not look like it is related to the test. And the test looks completed so you can just delete this.

Reply:
I think it is just another way of validating the same thing.